We do not have furo packaged and it is not needed to run tests.

--- elastic-transport-8.12.0/setup.py.orig
+++ elastic-transport-8.12.0/setup.py
@@ -69,7 +69,6 @@
             "aiohttp",
             # Override Read the Docs default (sphinx<2)
             "sphinx>2",
-            "furo",
             "sphinx-autodoc-typehints",
         ],
     },
